hack/update-generated-protobuf.sh

```diff
@@ -37,7 +37,13 @@ function prereqs() {
   KUBE_BUILD_CONTAINER_NAME="kube-build-${KUBE_ROOT_HASH}"
   KUBE_BUILD_DATA_CONTAINER_NAME="kube-build-data-${KUBE_ROOT_HASH}"
   DOCKER_MOUNT_ARGS=(
-    --volume "${REPO_DIR:-${KUBE_ROOT}}:/go/src/${KUBE_GO_PACKAGE}"
+    --volume "${REPO_DIR:-${KUBE_ROOT}}/cluster:/go/src/${KUBE_GO_PACKAGE}/cluster"
```

Comment:
In short, it's related to SELinux re-labeling happening to mount directory inside the container. All the directories/fies that are then created inside the container are owned by root, which causes problems @gmarek mentioned in his issue. The other possible approach is to build the binaries outside the container, then there's no problem (see my proposed solution for that: #25346.